packages/core-data/src/resolvers.js

```diff
@@ -480,6 +480,8 @@ export const getBlockPatterns =
 				switch ( key ) {
 					case 'block_types':
 						return 'blockTypes';
+					case 'post_types':
```

Comment:
Is there any documentation to update around this? cc @juanmaguitar

Reply:
Can we generalise the casing transformation?

```js
// hello_world -> helloWorld
function snakeToCamel( string ) {
  return string.replace( /_([a-z])/, ( _, letter ) => letter.toUpperCase() );
}

mapKeys( pattern, ( _, phpKey ) => snakeToCamel( phpKey ) );
```

If not (why not?), then it would be nice to eventually be able to source the [mapping from Core](https://github.com/WordPress/wordpress-develop/blob/e7cafe6141631bc8125c3173404edab09c868961/src/wp-includes/block-patterns.php#L201-L210) rather than keep a copy here.

Reply:
I applied the feedback, using a function from lodash that is already used in other parts of the codebase.